src/screens/Game/GameSubMenu/index.tsx

```diff
@@ -112,6 +115,26 @@ export default function GamesSubmenu({
     ipcRenderer.send('addShortcut', appName, runner, true)
   }
 
+  async function handleAddToSteam() {
+    setSteamRefresh(true)
+    await ipcRenderer.invoke('addToSteam', appName, runner)
+    isAddedToSteam()
+    setSteamRefresh(false)
+  }
+
+  async function handleRemoveFromSteam() {
+    setSteamRefresh(true)
+    await ipcRenderer.invoke('removeFromSteam', appName, runner)
+    isAddedToSteam()
+    setSteamRefresh(false)
+  }
+
+  const isAddedToSteam = () => {
+    ipcRenderer.invoke('isAddedToSteam', appName, runner).then((added) => {
+      setAddedToSteam(added)
+    })
+  }
```

Comment:
Maybe do:

```ts
  async function isAddedToSteam() {
    const added = await ipcRenderer.invoke('isAddedToSteam', appName, runner)
    setAddedToSteam(added)
  }
```

for consistency with the previous 2 functions?

Reply:
Good point. I rewrote everything in frontend to a more consistent and clean way